compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -948,6 +948,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool opCodeIsNoOpOnThisPlatform(TR::ILOpCode &opCode);
 
+   bool supportsPrefetch() { return getS390ProcessorInfo()->supportsArch(TR_S390ProcessorInfo::TR_z10); }
```

Comment:
I think the z/Architecture instructions this query is guarding are the two PREFETCH DATA (PFD and PFDRL).   The other forms of prefetch are named quite differently (branch preload for i-cache prefetching, NIAI for software hints on loading data).  If we want to improve it, I guess supportsPrefetchData() would be a more suitable name...